doc/src/manual/functions.md

```diff
@@ -24,26 +24,35 @@ both typing and visual noise.
 
 A function is called using the traditional parenthesis syntax:
 
-```julia
+```@meta
```

Comment:
Same remark as in https://github.com/JuliaLang/julia/pull/20183/files#r97537182 can apply here as well I think.